package.json

```diff
@@ -10,38 +10,38 @@
     "tslint-config-shopify",
     "shopify"
   ],
-  "main": "base.json",
+  "main": "./base.json",
```

Comment:
Just delete `base.json`, change this to `./base`, and the built `base.js` will be picked up.